src/YieldCalculator.js

```diff
@@ -0,0 +1,10 @@
+export class YieldCalculator {
+  static getYieldRate = (purchaseAmount, record) => {
+    let profit = 0;
+    record.forEach((table) => {
+      profit += table.count * table.priceNumber;
+    });
```

Comment:
```suggestion
    const profit = record.reduce((profit, table) => {
      profit += table.count * table.priceNumber;
      return profit;
    }, 0);
```
`reduce`를 사용하는건 어떨까요??

Reply:
저도 reduce 사용이 더 좋다고 생각합니다 :)